unified-prototype/unified-plugin/plugin-android/src/main/java/org/gradle/api/experimental/android/application/StandaloneAndroidApplicationPlugin.java

```diff
@@ -36,6 +36,10 @@ public void apply(Project project) {
         dslModel.getBuildTypes().getDebug().getMinify().getEnabled().convention(false);
         dslModel.getBuildTypes().getRelease().getMinify().getEnabled().convention(false);
 
+        // Enable desugaring automatically when JDK > 8 is targeted
+        dslModel.getCoreLibraryDesugaring().getEnabled().convention(project.provider(() -> dslModel.getJdkVersion().get() > 8));
+        dslModel.getCoreLibraryDesugaring().getLibVersion().convention("2.0.4");
```

Comment:
IMO the `2.0.4` version convention seems like more of a NIA-level convention

Reply:
It might be.  When conventions are available in the main branch, I'll move this back to the NiA project.  There are other conventions of SDK versions as well that probably need to move.